src/template-pages/guide-page.html

```diff
@@ -1,218 +0,0 @@
-<!DOCTYPE html>
```

Comment:
what replaced the guide-page?

Reply:
I placed the guide page just for testing so deleted after testing is completed.
Guide page is not a template page.